### ISSUE TITLE
Support docker create

### ISSUE THREAD

**Issue by soupdiver** [ansible/ansible-modules-core#906]
It would be nice to have support for the `docker create` command. This creates a container but does not starts it. This can be useful if further initialisation for the container is needed. I think the easiest way would be to introduce a state **created**
See: https://docs.docker.com/reference/commandline/cli/#create

**Comment by ansibot** [comment]
@cove, @softzilla, @zfil, @ThomasSteinbach, ping. This issue is waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@cove, @softzilla, @zfil, @ThomasSteinbach, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@soupdiver We appreciate you taking the time to file this issue, but the module you've opened the issue about has been deprecated.

We will consider critical bugfixes for deprecated modules, but all other types of requests will be closed.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)
